triton/examples/whisper/whisper.rst

```diff
@@ -0,0 +1,134 @@
+Whisper
+======
+
+This uses :doc:`Singularity containers </triton/usage/singularity>`,
+so you should refer to that page first for general information.
+
+There are two variants of Whisper available. The "standard" Whisper uses 
+whisper-ctranslate2, which is a CLI for faster-whisper, a reimplementation 
+of OpenAI's Whisper using Ctranslate2. Original repository for this 
+project can be found 
+`here <https://github.com/Softcatala/whisper-ctranslate2>`_.
+
+The second variant is whisper-diarization, which is a fork of faster-whisper 
+with support for speaker detection (diarization). 
+Original repository for this project can be found 
+`here <https://github.com/MahmoudAshraf97/whisper-diarization>`_.
+
+Of these two, whisper-diarization runs noticable slower and has less versatile 
+options. Using base Whisper is recommended if speaker detection is not necessary.
+
+Usage (Whisper)
+~~~~~
```

Comment:
The convention (notn always followed in older pages, though) is `=` for the page title, `-` for first-level subheadings, `~` for second level subheadings.  Not sure what's the convention after that...